packages/core/src/routes-me/user-assets.ts

```diff
@@ -57,13 +57,15 @@ export default function userAssetsRoutes<T extends AuthedMeRouter>(...[router]:
     '/user-assets',
     koaGuard({
       files: object({
-        file: uploadFileGuard,
+        file: uploadFileGuard.array().min(1),
       }),
       response: userAssetsGuard,
     }),
     async (ctx, next) => {
-      const { file } = ctx.guard.files;
+      const { file: bodyFiles } = ctx.guard.files;
 
+      const file = bodyFiles[0];
+      assertThat(file, 'guard.invalid_input');
```

Comment:
I think this assertion is duplicated with zod guard `.min(1)` in line 60.

Reply:
Discussed offline and let's move on. This assertion is to make TypeScript happy, since TypeScript has no context of the zod body guard